firmware/settings.py

```diff
@@ -93,4 +93,4 @@
 
 # Enable to run with Selenium
 # Set to the driver executable path
-#SELENIUM_DRIVER_EXECUTABLE_PATH = ''
+SELENIUM_DRIVER_EXECUTABLE_PATH = getcwd() + '/firmware/driver/geckodriver'
```

Comment:
About that:
Firstly, I'd prefer you not to store the driver in this repo. Rather you can add instructions how to download it to the README. This will save you some License concerns regarding the MPL 2.0 under which the geckodriver sources are licensed.
Secondly, you should use `pathlib`, specifically `pathlib.Path`, to do all things regarding path creation.

Reply:
Also the driver does not have the executable bit set, so this will not work out of the box

Reply:
Thanks for the comment, now the cloning party has to get the driver, which is preferred.